extensions/cornerstone/src/commandsModule.js

```diff
@@ -248,8 +248,8 @@ const commandsModule = ({ servicesManager }) => {
       if (enabledElement) {
         let viewport = cornerstone.getViewport(enabledElement);
         viewport.voi = {
```

Comment:
Find/replace error I guess. This won't work, cornerstone is expecting lowercase

Reply:
Oh interesting this one was hit by no tests, fixed the vtkjs one, thanks.